tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -46,7 +46,7 @@
    "outputs": [],
```

Comment:
Do another pass on the Overview section.
Few points:
- Consider writing in present simple and not future tense.
- In "Steps" - don't shorten the title: "Setup environment" -> "Setup the Environment", "Init dataset" -> "Initialize the dataset"...
- General rephrasing